src/com/sap/piper/analytics/Analytics.groovy

```diff
@@ -0,0 +1,106 @@
+package com.sap.piper.analytics
+
+import com.cloudbees.groovy.cps.NonCPS
+import org.jenkinsci.plugins.workflow.steps.MissingContextVariableException
+
+class Analytics implements Serializable{
+
+    private static Analytics instance
+
+    private List eventListeners
+
+    private Analytics(){
+        this.eventListeners = []
```

Comment:
Should we remote the *event* here as it's not related to the event type?

Reply:
yes, let's do this ...